xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -160,9 +199,10 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets a value that determines whether <see langword="null" /> values are ignored during serialization and deserialization. The default value is <see langword="false" />.</summary>
+        <value><see langword="true" /> to ignore null values during serialization and deserialization; otherwise, see langword="false" />.</value>
```

Comment:
This is missing a start tag
```suggestion
        <value><see langword="true" /> to ignore null values during serialization and deserialization; otherwise, <see langword="false" />.</value>
```